components/inputs/input-number.js

```diff
@@ -178,6 +179,7 @@ class InputNumber extends FocusMixin(LabelledMixin(SkeletonMixin(FormElementMixi
 			 * @ignore
 			 */
 			valueTrailingZeroes: { type: String, attribute: 'value-trailing-zeroes' },
+			_afterSlotWidth: { type: Number },
```

Comment:
Consumers don't need to set this but we do want changes to it to trigger a render, so it's a good use case for a reactive state property. The others here (`_hintType`, `_formattedValue`) should be the same but this code was written before state properties were a thing.

```suggestion
			_afterSlotWidth: { state: true },
```